css/Components/Title.scss

```diff
@@ -5,18 +5,26 @@ $Title-font-size: 72px;
 $Title-line-height: 91px;
 $Title-font-size-mobile: 39px;
 $Title-line-height-mobile: 52px;
-$Title-font-family: "MatrixIIOT";
-$Title-font-weight: 800;
+$Title-font-family: "matrix-ii";
+$Title-font-weight: bold;
 
 .Title {
-  color: $Color-madison;
+  color: $Color-white;
   font-size: $Title-font-size-mobile;
   line-height: $Title-line-height-mobile;
   font-family: $Title-font-family;
   font-weight: $Title-font-weight;
+  -webkit-font-smoothing: antialiased;
 
   @include Breakpoint-desktopOnly {
     font-size: $Title-font-size;
     line-height: $Title-line-height;
   }
 }
+
+.Title.Title--alternate {
+  color: transparent;
+  background-image: linear-gradient(118deg, $Color-coral, $Color-razzmatazz);
+  background-clip: text;
+  -webkit-background-clip: text;
```

Comment:
As I understand `-webkit-background-clip: text` is a proprietary value only for WebKit.